app/views/templates/project.html.slim

```diff
@@ -1,4 +1,7 @@
-.container-fluid
+.container-fluid ng-if='!project || !currentOrganization'
+  p.project-page-loading.loading
+    em Loading project data...
```

Comment:
This `p` tag probably doesn't look like the page header. It would cause less UI flickering to put this loading text into the `h1` in the page header.